### ISSUE TITLE
Error when exporting PDF with a list of authors

### ISSUE THREAD

**Issue by osnofa** [l1xnan/obsidian-better-export-pdf#357]
I get the following error when the 'author' property is of type 'List':
`TypeError: 'author' must be of type 'string', but was actually of type 'NaN'`

This became relevant after the Obsidian Web Clipper browser extension was released, since the default behaviour for 'author' when saving webpages is to generate a 'List' instead of a 'Text'.

Maybe a quick patch would be to get the first author when the property is a list.

**Comment by Scavy** [comment]
This has already been fixed in the unreleased code for issue #328
I guess it will be available in next release.

**Comment by Scavy** [comment]
If you dare to do it yourself, you can replace the following code in the plugin files:
.plugins/better-export-pdf/main.js

Line 20139:
`    pdfDoc.setAuthor(author);`

Replace with this:
```
    if (Array.isArray(author)) {
      pdfDoc.setAuthor(author.join(", "));
    } else {
      pdfDoc.setAuthor(author.toString());
    }
```

This checks if the author is an array instead of a string, and if so, it joins all the values into a comma separated string.
